apps/web/src/routes/_content/courses/$courseId/$chapterId.tsx

```diff
@@ -799,7 +799,7 @@ function CourseChapter() {
             </div>
 
             <div className="flex w-full flex-col items-center justify-center lg:max-w-[1102px] lg:flex-row lg:items-stretch lg:justify-stretch">
-              <div className="text-blue-1000 w-full space-y-4 break-words px-4 md:px-2 md:mt-8 md:grow md:space-y-6 md:overflow-hidden pb-2">
+              <div className="text-blue-1000 max-w-[796px] w-full space-y-4 break-words px-4 md:px-2 md:mt-8 md:grow md:space-y-6 md:overflow-hidden pb-2">
```

Comment:
this breaks final exam layout
add the class conditionally instead (!chapter.isCourseExam)